java/src/jmri/jmrit/beantable/signalmast/SignalMastTableDataModel.java

```diff
@@ -730,12 +452,12 @@ JComboBox getEditorBox(int row) {
     }
     Hashtable<Object, JComboBox> editorMap = new Hashtable<Object, JComboBox>();
 
-    /**
+/*    *//**
```

Comment:
It was created by the IDE when I commented out the getRendererBox() method and Hashtable